lib/events/complete.go

```diff
@@ -51,9 +51,6 @@ type UploadCompleterConfig struct {
 	CheckPeriod time.Duration
 	// Clock is used to override clock in tests
 	Clock clockwork.Clock
-	// Unstarted does not start automatic goroutine,
-	// is useful when completer is embedded in another function
-	Unstarted bool
```

Comment:
Why get rid of `Unstarted`?

If we're going to do this, I wonder if we should unexport `CheckUploads`. I don't like the idea of an API that allows you to run checks manually which may conflict with checks that are already happening in the background.

Reply:
`newUploadCompleter` can be used to start it without starting now. `checkUploads` is no longer exported. Both of these are used in tests.